.github/workflows/main.yml

```diff
@@ -15,16 +15,15 @@ on:
 jobs:
   main:
     runs-on: ubuntu-latest
-
+    permissions:
+      packages: write
+      contents: read
     steps:
       - uses: actions/checkout@v1
       - name: Set up JDK 1.8
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: lint
-        # Run ./gradlew spotlessApply when failing on this step.
-        run: ./gradlew spotlessCheck
```

Comment:
Remove spotlessCheck. because spotless requires Gradle 5.4 or newer, this was 4.1.

Reply:
Is it possible to update gradle?
```
 ./gradlew wrapper --gradle-version 6.9.1
```

Reply:
Thanks for your comment.
I tried to run above command and updating gradle version was successful. [865b017](https://github.com/trocco-io/embulk-input-facebook_ads_insights/pull/20/commits/865b01784c80a916fa9d5f5759d57193f4a6524f)
I add `./gradlew spotlessCheck` step again.